pandas/io/packers.py

```diff
@@ -506,11 +526,17 @@ def decode(obj):
         dtype = dtype_for(obj[u'dtype'])
         pd_dtype = pandas_dtype(dtype)
         np_dtype = pandas_dtype(dtype).base
+
+        ctor_dtype = np_dtype
+        if is_categorical_dtype(pd_dtype):
+            # Series ctor doesn't take dtype with categorical
+            ctor_dtype = None
```

Comment:
it already does.
However in this case want to construct a ``pd.Categorical`` as you can pass all of the attributes thru
```
In [1]: Series(list('aabc'),dtype='category')
Out[1]: 
0    a
1    a
2    b
3    c
dtype: category
Categories (3, object): [a, b, c]
```